pandas/tests/io/test_common.py

```diff
@@ -351,6 +350,8 @@ def test_read_fspath_all(self, reader, module, path, datapath):
         ],
     )
     def test_write_fspath_all(self, writer_name, writer_kwargs, module):
+        if writer_name in ["to_latex"]:  # uses Styler implementation
+            pytest.importorskip("jinja2")
```

Comment:
So this new implementation will require `jinja2` to be installed first?

Reply:
correct. @jreback was fine for this to be the case.

Reply:
No objection in having that as a requirement, but we regard this as a breaking change, yes?

Reply:
it is a breaking chang, yes. Current implemtations of dataframe.to_latex will fail after this PR if the user does not have jinja2.

fallback option: since the pr does not remove the LatexFormatter, code could be redirected on undetected jinja2. this would then be non-breaking.

Reply:
If the fallback behavior was implemented then I think this PR would be okay for 1.5. If not, this could be one of the "breaking without deprecation" behaviors for 2.0 possibly if this change is "too large" https://github.com/pandas-dev/pandas/issues/44823

Reply:
When thinking about doing this I realised the tests that I have altered will be a nightmare for dual implementation, so if this needs to go to 2.0, I don't oppose that.

But then for 2.0 I will probably look to change the arg signature of all of this anyway.

Reply:
What would the args be changed to? The current arguments should be deprecated.